boards/arm/cc1352p1_launchxl/cc1352p1_launchxl.dts

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2021 Florin Stancu
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <ti/cc1352r.dtsi>
+#include "../cc1352r1_launchxl/boosterpack_connector.dtsi"
+#include "../cc1352r1_launchxl/cc1352r1_launchxl-pinctrl.dtsi"
+
+#define BTN_GPIO_FLAGS (GPIO_ACTIVE_LOW | GPIO_PULL_UP)
```

Comment:
we normally don't do this, just add the appropriate flags to the gpio flags cell.